test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -49,18 +49,19 @@ class cursor_bound_01 : public test {
     /* Class helper to represent the lower and uppers bounds for the range cursor. */
     class bound {
         public:
-        bound() = default;
-        bound(uint64_t key_size_max, bool lower_bound, std::string key)
-            : _lower_bound(lower_bound), _key(key)
+        bound()
+        {
+            clear();
+        };
+
+        bound(uint64_t key_size_max, bool lower_bound) : _lower_bound(lower_bound)
         {
             bool set_inclusive = random_generator::instance().generate_integer(0, 1);
-            // FIXME: Use random strings, once bounds are implemented properly.
-            // auto key_size =
-            //   random_generator::instance().generate_integer(static_cast<uint64_t>(1),
-            //   key_size_max);
-            // auto random_key = random_generator::instance().generate_random_string(
-            //   key_size, characters_type::ALPHABET);
-            // _key = random_key;
+            auto key_size =
+              random_generator::instance().generate_integer(static_cast<uint64_t>(1), key_size_max);
+            auto random_key = random_generator::instance().generate_random_string(
```

Comment:
The variable `random_key` can be removed, we can do this instead:
```
_key = random_generator::instance().generate_random_string(key_size, characters_type::ALPHABET);
```
Same for `set_inclusive`:
```
_inclusive = random_generator::instance().generate_integer(0, 1);
```